pkg/utils/utils.go

```diff
@@ -693,3 +699,42 @@ func isSwitchdev(name string) bool {
 
 	return true
 }
+
+// IsKernelLockdownMode returns true when kernel lockdown mode is enabled
+func IsKernelLockdownMode() bool {
+	out, err := RunCommand("cat", "/host/sys/kernel/security/lockdown")
+	glog.V(2).Infof("IsKernelLockdownMode(): %s, %+v", out, err)
+	if err != nil && strings.Contains(out, "[integrity]") {
```

Comment:
Is there a link describing the possible values of `lockdown` file ?

I am a bit confused,
if err is not nil why are we checking the content of `out` ? also L710 hints that `[integrity]` value means we are indeed in kernel lockdown so i dont understand the second expression in this if statement

Reply:
Fixed

No description afaik, code [here](https://github.com/torvalds/linux/blob/master/security/lockdown/lockdown.c#L42)